mypy/test/testcmdline.py

```diff
@@ -26,6 +26,14 @@
     'reports.test',
 ]
 
+# Runs the test in the specified working directory
+custom_working_dir = {
+    "testFailedImportCanBeFoundInParent": "main/subdir",
+    "testFailedImportCannotBeFoundInParent": "main/subdir",
+    "testImportInParentButNoInit": "main/subdir",
+    "testFailedImportCanBeFoundInParentsParent": "main/subdir1/subdir2",
+}
```

Comment:
I'd prefer having a way of specifying this in the test case, as this is more fragile (consider what happens if one of the test cases is renamed). For example, this could be specified via `# cwd: main/subdir` in the test case description, similar to how we have `# cmd: ...` and similar.  This would also make it easier to understand what the test cases are testing exactly.

This is not absolutely essential, but it would be nice to have. If you decide not to implement this, please add a comment to each test case that describes the working directory and mentions `custom_working_dir`.

Reply:
Added `# cwd: ...` to `testcmdline.py` 👍